cdisc_rules_engine/rules_engine.py

```diff
@@ -173,6 +177,8 @@ def validate_single_rule(
             error_obj: ValidationErrorContainer = self.handle_validation_exceptions(
                 e, dataset_path, dataset_path
             )
+            if self.is_validate_xml:
```

Comment:
This line is breaking validation. We should not set a rule to success if it has an execution error. I believe what we want to do instead is just raise an error during validation if the is_validate_xml is set to true.